swagger/swagger-generator/generator-core/src/main/java/org/apache/servicecomb/swagger/extend/annotations/RawJsonRequestBody.java

```diff
@@ -23,16 +23,14 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-import org.springframework.core.annotation.AliasFor;
-
 @Target({ElementType.PARAMETER})
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
 public @interface RawJsonRequestBody {
-  @AliasFor("name")
+  // aliasFor "name"
```

Comment:
Will it introduce some backward compatibility issue?

Reply:
will not, because we read both of the two field